index.html

```diff
@@ -141,6 +141,19 @@ <h4>Daten</h4>
                                                 </div>
                                                 <div class="small-9 columns">
                                                     <input type="checkbox" name="form-distribution-equally" value="1" checked="checked" id="form-distribution-equally"><label for="form-distribution-equally">Arbeitszeit gleichmäßig auf Tage aufteilen</label>
+                                                    <input type="checkbox" name="form-autofill-employee-date" value="0" id="form-autofill-employee-date"><label for="form-autofill-employee-date">Datum der Unterschrift vorausfüllen</label>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="row">
+                                        <div class="small-11">
+                                            <div class="row">
+                                                <div class="small-3 columns">
+                                                    <label class="right">Unterschrift hinzufügen (optional)</label>
```

Comment:
```suggestion
                                                    <label class="right" for="form-autofill-employee-signature">Unterschrift hinzufügen (optional)</label>
```

Reply:
Adding the for-attribute to the label will focus the upload field whenever a user clicks on the text instead of the input field.